test/test_cuda_mapper.cc

```diff
@@ -753,6 +753,38 @@ def perforatedConvolution(float(N, C, H, W) input, float(M, C, KH, KW) weights,
   Prepare(tc);
 }
 
+TEST_F(PolyhedralMapperTest, ModulusConstantRHS) {
+  string tc = R"TC(
+def fun(float(N) a) -> (b) { b(i) = a(i % 2) where i in 0:N }
+)TC";
+  // This triggers tc2halide conversion and should not throw.
+  auto scop = Prepare(tc);
+  for (auto r : scop->reads.range().get_set_list()) {
+    // skip irrelevant reads, if any
+    if (r.get_tuple_name() != std::string("a")) {
+      continue;
+    }
+    // EXPECT_EQ(r.get_stride(?), ?);
```

Comment:
I am not quite sure what should be the `pos` for `get_stride()`.

Reply:
index of the dimension in the set (position of the subscript that should involve the stride), `0` in this case a believe